src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -34,10 +34,10 @@
     <a href="/conversations">Conversations</a>
     <% if(request.getSession().getAttribute("user") != null){ %>
       <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-      <a href="/profile">Profile</a>
     <% } else{ %>
       <a href="/login">Login</a>
     <% } %>
+    <a href="/profile">Profile</a>
```

Comment:
I think we want to keep this inside this if statement, so that it only shows when the user is logged in.

Reply:
I see! This was my initial intention!! Thanks